examples/stm32/f4/stm32f429i-discovery/ts-i2c/gfx.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * A simple port of the AdaFruit minimal graphics code to my
+ * demo code.
+ */
```

Comment:
file is ad verbatim copy of chucks gfx.h in lcd-serial example. do we modify both? if so, chuck as copyright holder?